processor/src/main/java/com/fourlastor/pickle/TestClass.kt

```diff
@@ -10,5 +11,5 @@ data class TestField(val type: TypeElement) {
             get() = type.qualifiedName.toString().decapitalize().replace('.', '_')
 }
 
-fun testMethodStatement(field: TestField, statementFormat: String, vararg args: Any) =
-        TestMethodStatement(field, statementFormat, args.asList())
+fun testMethodStatement(testField: TestField, statementFormat: String, methodName: Name, vararg args: Any) =
```

Comment:
I thought using `args` any to collect fieldName, methodName and actual args all together was confusing. Now it is slightly easier to understand. Let me know what you think.